pkg/network/protocols/http/monitor.go

```diff
@@ -94,6 +96,8 @@ func NewMonitor(c *config.Config, offsets []manager.ConstantEditor, sockFD *ebpf
 		return nil, fmt.Errorf("couldn't instantiate batch manager: %w", err)
 	}
 
+	processMonitor := monitor.GetProcessMonitor()
```

Comment:
return error if failed to get a process monitor (returned `nil` from `GetProcessMonitor`)